### ISSUE TITLE
Some questions regarding the paper

### ISSUE THREAD

**Issue by Divadi** [kamse/CostDCNet#2]
Thank you for the interesting work! I just had a quick question regarding the Ablation study in Table 4.
![image](https://user-images.githubusercontent.com/17109672/204106424-d493807e-ad13-464e-ad51-939edcf65bea.png)

Going from Row 1 to Row 2, the 2D (RGBD) UNet is replaced with the proposed 3D UNet. I was wondering - how does the # of parameters decrease when going from 2D to 3D UNet? I would have thought that going from 2D to 3D convolutions, the # of parameters would increase. More specifically, what is the structure of the 2D UNet? And, am I correct in assuming that the loss for the 2D UNet is simply direct regression + L1 instead of soft-argmax + L1?

Also, before the 2D & 3D encoders are added, when generating the fused volume, what exactly are the 2D and 3D features used?

**Comment by kamse** [comment]
Thank you for your interest.

Frist, we used the basic 2D Unet model (link: https://github.com/milesial/Pytorch-UNet). 
It has more levels and channels compared to our 3D Unet.

Second, the loss for the 2D UNet is also soft-argmax + L1.
To do this, the output of the 2D unet is cost maps (# of depth plane x width x height).

**Comment by Divadi** [comment]
I see, thank you for your clarification. Do you plan to make the training code available soon?
